tests/sls/nginx/headers.py

```diff
@@ -0,0 +1,30 @@
+"""
+Test nginx's security related headers
+
+Test the headers very naively - their existence and value. Only check the
+top-most '/' domain.
+"""
+import urllib.request
+
+from tests.util import Failure, Success
+
+
+def run():
+    expected_headers = [
+        ('Content-Security-Policy',
+            "default-src 'self'; frame-ancestors 'none'"),
+        ('X-Frame-Options', 'DENY'),
+        ('X-XSS-Protection', '1; mode=block'),
+        ('X-Content-Type-Options', 'nosniff')]
```

Comment:
style nit: leave a trailing comma after this last tuple, and put the closing `]` by itself on the next line.